### ISSUE TITLE
The log should end with a mention of warnings if they occured

### ISSUE THREAD

**Issue by Kissaki** [davetcc/hugojoomla#13]
I just noticed the conversion process logs a lot of warnings. This is not visible from the last few lines that imply a success with no mention of warnings:

> Finished conversion of Joomla database

I would at least expect a hint that warnings occured and should be checked in the conversion log (prior log messages).

**Comment by davetcc** [comment]
Agreed. failure state should be captured and output. At least to indicate the conversion wasn’t completely successful.
